### PR TITLE
Exercise 4

### PR DESCRIPTION
**Pull request by jpatel386**
Please make sure you are raising this PR against your own repository and not the original. If it says https://github.com/CorndelWithSoftwire in the search bar right now, you are in the wrong place!

Please also don't forget to submit the PR's url to Aptem afterwards (by clicking on the corresponding exercise component, pasting the url in the textbox that appears and then pressing the finish button).

### CODE REVIEW

**Comment by jpatel386** [comment]
Ignore changes to all other files. Need to merge exercise-3 branch still. Only need to look at vagrant file

File: Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,47 @@
+Vagrant.configure("2") do |config|
+  config.vm.box = "hashicorp/bionic64"
+
+  config.vm.provision "shell", privileged: false, inline: <<-SHELL
+
+  sudo apt-get update
```
Comment (hugh-emerson) [inline Vagrantfile]: I would indent this section up to `SHELL` as it's just the value for the `inline` argument for the `config.vm.provision` function on line 4

File: Vagrantfile [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+  # TODO: Install pyenv prerequisites
```
Comment (hugh-emerson) [inline Vagrantfile]: No longer TODO

**Review by hugh-emerson — APPROVED** [review]
Looks good. This is the kind of thing worth documenting in the README (e.g. you can run the app via a Vagrant VM if you ...)
But merge when you're ready 👍

**Pull request merged**